### DISCUSSION TITLE
Fix two-finger scrolling on laptop touchpads on single page view.

### DISCUSSION THREAD

**Discussion by moehammm** [sumatrapdfreader/sumatrapdf#4469] [Ideas]
When using the single-page view, scrolling with two fingers down or up (on Dell XPS laptop, _I can't say for other laptops_) it jumps too far between pages (ie goes from page 1 to 40). This issue does not exist on multiple page view. 
My idea is to reassign the between pages scrolling by two-finger swiping of the pad to left/ right.
Or make the scrolling jump one page at a time because it jumps 20 pages with a single slight stroke. 

Best application in existence by the way. 
Thanks for it and thanks in advance.

**Comment by kjk** [comment]
It's already fixed in [pre-release](https://www.sumatrapdfreader.org/prerelease)

**Reply by moehammm** [reply]
THANK YOU!! 
one little tiny request if possible, can you make it lock on to the page (prevent scrolling) when zoomed in?
